EasyCommands/BlockHandlers/LightBlockHandlers.cs

```diff
@@ -19,10 +19,39 @@
 
 namespace IngameScript {
     partial class Program {
+        public class SearchlightHandler : SubTypedBlockHandler<IMyFunctionalBlock> {
+            Dictionary<String, int> targetingGroupLookup = NewDictionary(
+                KeyValuePair("Default", 0),
+                KeyValuePair("Power", 1),
+                KeyValuePair("Weapons", 2),
+                KeyValuePair("Propulsion", 3));
```

Comment:
This dictionary belongs somewhere else, since it will be needed for all turreted gun.

Reply:
Whoops, ya it isn't needed until the target options support is added.